src/components/Navigation/Navigation.scss

```diff
@@ -208,14 +208,14 @@ $disabled-fade: 0.6;
     @include recolor-icon(
       var(--p-icon-hovered),
       transparent,
-      $filter-color: filter('action')
+      $filter-color: var(--p-action-primary)
     );
     background: var(--p-background-hovered);
 
     @media (-ms-high-contrast: active) {
       @include recolor-icon(
         $fill-color: var(--p-icon-on-interactive),
-        $filter-color: filter('white')
+        $filter-color: var(--p-icon-on-interactive)
```

Comment:
These filter-color values passed to the 3rd argument of recolor-icon  are sent as the [`filter`](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) property. They're not actual color values, they're filters like [`brightness(0) saturate(100%) invert(100%)`](https://github.com/Shopify/polaris-tokens/blob/main/dist/color-filters.color-map.scss#L138).

I think changing these to colors is going to break the behaviour, which is to allow for the recoloring of icons that are passed as svg datauris and rendered with an img tag